.github/workflows/csi_operator_ci.yaml

```diff
@@ -0,0 +1,71 @@
+name: Community Operators test
+on:
+  push:
+    branches:
+      - task/CSI-2892_cert_community_operator_hub
```

Comment:
is this for the target branch, private branch, or both?
what should be the value here before this PR merge?